tdrs-backend/tdpservice/settings/cloudgov.py

```diff
@@ -120,6 +122,26 @@ class CloudGov(Common):
         "Cache-Control": "max-age=86400, s-maxage=86400, must-revalidate",
     }
 
+    AWS_ELASTIC_ACCESS_KEY = os.getenv('AWS_ELASTIC_ACCESS_KEY', '')
+    AWS_ELASTIC_SECRET = os.getenv('AWS_ELASTIC_SECRET', '')
+
+    awsauth = AWS4Auth(
+        AWS_ELASTIC_ACCESS_KEY,
+        AWS_ELASTIC_SECRET,
+        'us-gov-west-1',
+        'es'
+    )
+
+    # Elastic
+    ELASTICSEARCH_DSL = {
+        'default': {
+            'hosts': os.getenv('ELASTIC_HOST', ''),
```

Comment:
likewise, `ELASTIC_HOST` is provided after binding the service, and needs to be added to the applications environment variables